great_expectations/core/validation_config.py

```diff
@@ -222,3 +222,13 @@ def run(
             result_format=result_format,
         )
         return validator.validate_expectation_suite(self.suite, evaluation_parameters)
+
+    @public_api
+    def save(self) -> None:
+        store = project_manager.get_validation_config_store()
```

Comment:
Didn't we say we want the pattern here to be that we get the store upon initialization? (e.g. that's what we do for expectation suites). Both for raising earlier, and for predictability in case someone switches contexts. I think we could hijack a root_validator for this, even though I don't love it.